lib/signalfx/tracing/instrumentation/sidekiq.rb

```diff
@@ -0,0 +1,35 @@
+module SignalFx
+  module Tracing
+    module Instrumenter
+      module Sidekiq
+
+        Register.add_lib :Sidekiq, self
+
+        class << self
+          def instrument(opts = {})
+            return if @instrumented
+            begin
+              require 'sidekiq'
+            rescue LoadError
```

Comment:
ruby noob Q: why `LoadError` below is outputting the error message but not this one?

Reply:
The procedure is to check if a relevant target library is accessible and return as a noop if not.  If one is, but the applicable instrumentation hasn't been installed a warning will be printed so that lack of traced activity isn't unexpected.